src/DayContainer.js

```diff
@@ -0,0 +1,107 @@
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        :
+        props.day === 'Tuesday'
+        ?
+        <div className="event__container">
+          <li>
+            <h2 className={`event__title--${props.day}`}>{props.day}</h2>
+            <ul>
+              <li className={`event__list--${props.event}`}>{props.event}</li>
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        :
+        props.day === 'Wednesday'
+        ?
+        <div className="event__container">
+          <li>
+            <h2 className={`event__title--${props.day}`}>{props.day}</h2>
+            <ul>
+              <li className={`event__list--${props.event}`}>{props.event}</li>
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        :
+        props.day === 'Thursday'
+        ?
+        <div className="event__container">
+          <li>
+            <h2 className={`event__title--${props.day}`}>{props.day}</h2>
+            <ul>
+              <li className={`event__list--${props.event}`}>{props.event}</li>
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        :
+        props.day === 'Tuesday'
+        ?
+        <div className="event__container">
+          <li>
+            <h2 className={`event__title--${props.day}`}>{props.day}</h2>
+            <ul>
+              <li className={`event__list--${props.event}`}>{props.event}</li>
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        :
+        props.day === 'Friday'
+        ?
+        <div className="event__container">
+          <li>
+            <h2 className={`event__title--${props.day}`}>{props.day}</h2>
+            <ul>
+              <li className={`event__list--${props.event}`}>{props.event}</li>
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        :
+        props.day === 'Saturday'
+        ?
+        <div className="event__container">
+          <li>
+            <h2 className={`event__title--${props.day}`}>{props.day}</h2>
+            <ul>
+              <li className={`event__list--${props.event}`}>{props.event}</li>
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        :
+        props.day === 'Sunday'
+        ?
+        <div className="event__container">
+          <li>
+            <h2 className={`event__title--${props.day}`}>{props.day}</h2>
+            <ul>
+              <li className={`event__list--${props.event}`}>{props.event}</li>
+              <p className='event__detail'>{props.detail}</p>
+            </ul>
+          </li>
+        </div>
+        : null
```

Comment:
Each of these conditional renders appears to be identical. Could this be reduced to a condition that simply checks if props.day exists: render if true, null if false?
If you do need to make sure that the prop contains a weekday and not some other random text, you could verify this in a helper function or even in the single ternary.

Let me know if you'd like me to clarify.